### ISSUE TITLE
Failed to create EWS trace listener

### ISSUE THREAD

**Issue by BenjaminRamsay** [David-Barrett-MS/PowerShell-EWS-Scripts#63]
Hi David, first off you are a living legend for creating and sharing this project. Thank you so much for putting your time and effort into it. It's incredibly helpful. 

I'm using the Merge-MailboxFolder script, and getting the following error:
```
Cannot find an overload for "new" and the argument count: "1".
At C:\Users\benja\source\PowerShell\PowerShell-EWS-Scripts\Merge-MailboxFolder.ps1:951 char:9
+         $script:Tracer=[EWSTracer]::new($TraceFile)
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodException
    + FullyQualifiedErrorId : MethodCountCouldNotFindBest
 
12/13/2024 3:56:03 PM [CreateService] Failed to create EWS trace listener.  Throttling back-off time won't be detected.
```

My call looks like this:
`.\Merge-MailboxFolder.ps1 -SourceMailbox $mailbox -TargetArchive -MergeFolderList @{"Deleted Items" = "WellKnownFolderName.DeletedItems"} -OnlyItemsSentReceivedBefore $beforeDate -Office365 -OAuth -OAuthTenantId $tenantId -OAuthClientId $clientId `

The script continues to run after the error, and works fine as long as the number of items to process is small. But if there are a lot we start running into throttling issues, I'm guessing because of the error above.

Any ideas?

Thank you again!

**Comment by BenjaminRamsay** [comment]
Disregard, I found the culprit. I had trouble using the compiled copy of the EWS API dll that you provided, so I was allowing the script to search for another copy in Program Files. It found one installed with SQL Server Management Studio. For whatever reason, that copy of the DLL caused this error.

I fixed my original issue and used your copy of the DLL, and this error went away.

In case it helps others, my original problem using the DLL you provided was this error:
`[ErrorReported] ERROR (LoadLibraries): Could not load file or assembly '<path>\Microsoft.Exchange.WebServices.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)`

I knew to go into the file properties of the DLL and "Unblock" it, and had already done so with no success. But I finally realized you have to close and reopen PowerShell before it realizes the file is now unblocked.
